includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -440,14 +440,13 @@ protected function has_dev_mode_exemption( DOMNode $node ) {
 	/**
 	 * Check whether a certain node should be exempt from validation.
 	 *
-	 * @deprecated Use AmpProject\DevMode::isExemptFromValidation( $node ) instead.
+	 * @since 1.5 Exempts DOMNode if the 'data-ampdevmode' attribute has the value '_amp_exempt'.
```

Comment:
Can I un-deprecate this method, or should it stay and create a new function instead?

Reply:
I think this can be reverted based on my comment on the `sanitize` method below.